t/page/country.rb

```diff
@@ -0,0 +1,29 @@
+require 'minitest/autorun'
+require_relative '../../lib/page/country'
+require 'pry'
+
+describe 'Country' do
+  let(:all_countries) do
+    [
+      {
+        name: 'Abkhazia',
+        url:  'abkhazia',
+      },
+    ]
+  end
+
+  it 'detects that a country is not missing' do
```

Comment:
This seems like a slightly odd primary purpose for the test. There's already a test below for handling the case where the country doesn't exist — this one seems like it should be about the normal case where you do get a country, and can then access something on that that's needed in the template (e.g. its legislatures)